docs/development.md

```diff
@@ -142,21 +144,13 @@ It is recommended to use Firefox 46.0 as this is the browser used in CI build (T
 
 * If you want to use a Firefox version other than your computer's default, specify the custom path in `test.firefox.path` value in `test.properties`.
 
-* If you are planning to test changes to JavaScript code, disable JavaScript caching for Firefox:
-  * Enter `about:config` into the Firefox address bar and set `network.http.use-cache` (or `browser.cache.disk.enable` in newer versions of Firefox) to `false`.
-
```

Comment:
On second thoughts, can leave the two paragraphs in. I realised they are about manual (not automated) testing 😅